src/actions/index.js

```diff
@@ -376,6 +378,14 @@ export function removeCommunityModerator (community, moderatorId, prevProps) {
   }
 }
 
+export function removeCommunityMember (community, userId, cacheId, prevProps) {
+  return {
+    type: REMOVE_COMMUNITY_MEMBER,
+    payload: {api: true, path: `/noo/community/${community.id}/member/${userId}`, method: 'DELETE'},
+    meta: {slug: community.slug, userId, cacheId, prevProps}
```

Comment:
it feels wrong to me to have this action depend on cacheId and prevProps -- it's really the reducer that should figure out which caches have this member in them and remove the member's id.

for example, if i load the members page, and the person i'm looking to remove is somewhere on the page, but i don't see it, i might search for their name. so now in cache we have `subject=community&id=foo` and `subject=community&id=foo&search=tim`. if i call `removeCommunityMember` now, it should remove it from both caches.

the presence of `prevProps` complicates things... i think it's too coupled now to do optimistic updating this way. i'm not sure how else to do it (just spent a few minutes thinking about it, didn't come up with a clear answer), but perhaps we should just do things non-optimistically for now until we have time to come up with a better solution.